README.md

```diff
@@ -119,7 +119,7 @@ To import pod you should add string:
 - For `Swift` projects:
 
 
-        import CoreActionSheetPicker
+        import ActionSheetPicker_3_0

```

Comment:
@skywinder actually, I get an error like `no such module named CoreActionSheetPicker` when i `import CoreActionSheetPicker`.
`import ActionSheetPicker_3_0` work fine with Xcode7.3 and swift2.2 for me...